spec/features/site_customization/custom_pages_spec.rb

```diff
@@ -145,10 +137,9 @@
 
       scenario "Not listed in more information page due to different locale" do
```

Comment:
I don't think this test makes much sense anymore, with the new multi-language implementation.

It would be good, however, to have an example testing the new implementation (i.e. create a multi-language page, and test that the same page is rendered in different languages when the application locale changes). 

You could put it in a new `"Translation"` context.

Reply:
Hi @mlovic , Ok I will remove that scenario. About the new test, I already implemented in spec/features/translations_spec.rb in "Pages" context. Is right like that or you want it in spec/features/site_customization/custom_pages_spec.rb in "Translation" context. ?

Reply:
Oh right, I missed that! Yes, that's exactly what I meant.

The only thing is that, as part of the refactor, we extracted those tests into RSpec shared examples  (see [here](https://github.com/consul/consul/commit/612fdb09dddeb24c044e88fb9678c6833be1b393#diff-d4be06c13a93469215eda200da2e466fR53)), but that only covers the admin side of the application. It won't test the part of the application that a normal user sees. This means that we'll lose that part of the test when you adapt to the refactored code. 

If you can add it as an example to `spec/features/site_customization/custom_pages_spec.rb`, that would be great!

Let me know if anything doesn't make sense. Thank you!

Reply:
Ah ok! So you mean that you guys delete the file spec/features/translations_spec.rb. Ok no problem I will put the tests in spec/features/site_customization/custom_pages_spec.rb. Clear!